### DISCUSSION TITLE
Can renovate use GitHub releases API for GitHub action updates?

### DISCUSSION THREAD

**Discussion by ReenigneArcher** [renovatebot/renovate#29248] [Request Help]
### What would you like help with?

I would like help with my configuration

### How are you running Renovate?

Mend Renovate hosted app on github.com

### Please tell us more about your question or problem

I am getting ready to implement renovate as a replacement for dependabot.

One issue I'm facing with dependabot is that it doesn't check the GitHub releases API, and can therefore update an action which was not actually released. For example, it may be a draft or pre-release but the tag was already created.

From my quick reading, it seems the behavior would be the same with renovate, as it seems to only consider tags? https://docs.renovatebot.com/modules/manager/github-actions/#supported-datasources

### Logs (if relevant)

<details><summary>Logs</summary>

```

Replace this text with your logs, between the starting and ending triple backticks

```

</details>

**Comment by rarkins** [comment]
Yes, Renovate uses github-tags so unreleased tags would also work. In fact we really should be using `refs` because GitHub Actions also supports branch names too

**Reply by ReenigneArcher** [reply]
Thanks for the response... is there anyway to ignore tags that are in pre-release state?

I don't want to update to these versions.
![image](https://github.com/renovatebot/renovate/assets/42013603/a9d3f885-25c2-4332-aa44-adfc670304fa)

**Reply by rarkins** [reply]
I think these might already be classified as "unstable" and therefore skipped by Renovate. You can create a test repository and install the hosted bot to verify these things easily
